src/adapters/postgres/user-adapter.ts

```
@@ -972,6 +976,8 @@ export class PostgresUserService implements IServicelocator {
         'deviceId',
         'mobile_country_code',
         'country',
+        'permanentCountry',
```

ESLint

[error] Replace `'permanentCountry'` with `"permanentCountry"`

(prettier/prettier)